source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -57,3 +57,51 @@ todosController.get('remaining');
 
 Note that `@each` only works one level deep. You cannot use nested forms like
 `todos.@each.owner.name` or `todos.@each.owner.@each.name`.
+
+Sometimes, you may only care if the items of an array have changed, rather than their specific properties.
+For example, items may have been added to or removed from the array. In this case, you can create a
+computed property that relies on the leaf node:
+
+
+```app/controllers/todos.js
+export default Ember.Controller.extend({
+  todos: [
+    Ember.Object.create({ isDone: true }),
+    Ember.Object.create({ isDone: false }),
+    Ember.Object.create({ isDone: true })
+  ],
+
+  selectedTodo: null,
+  indexOfSelectedTodo: Ember.computed('selectedTodo', 'todos.[]', function () {
+    return this.get('todos').indexOf(this.get('selectedTodo'));
+  })
+});
+```
+
+Note here that we use the special key `[]` instead of `@each`, which is used for observing 
+a specific dependent key on each item. This instructs Ember.js to only care about the array 
+itself having items added, removed, or replaced, rather than changes to properties on the items themselves.
+
+Several of the [Ember.computed](http://emberjs.com/api/classes/Ember.computed.html) macros 
+utilize the `[]` key to implement common use-cases as well. For example, if you wanted to 
+create a computed property that mapped properties from an array, you could use 
+[Ember.computed.map](http://emberjs.com/api/classes/Ember.computed.html#method_map)
+or build the computed yourself:
+
+```javascript
+const Hamster = Ember.Object.extend({
+  excitingChores: Ember.computed('chores.[]', function () {
+    return this.get('chores').map(function (chore, index) {
+      return `CHORE ${index}: ${chore.toUpperCase()}!`;
+    }
+  })
+});
+
+const hamster = Hamster.create({
+  chores: ['clean', 'write more unit tests']
+});
+
+hamster.get('excitingChores'); // ['CHORE 1: CLEAN!', 'CHORE 2: WRITE MORE UNIT TESTS!']
+hamster.get('excitingChores').push('review code');
```

Comment:
@rwjblue @locks I know that we augment native arrays, but shouldn't `pushObject` be listed in the API docs if that's a method we should be using? I'm loathe to include example code using methods that aren't even documented.